cypress/e2e/settings-component.cy.js

```diff
@@ -1,68 +1,62 @@
-describe("Tests for settings component", () => {
+describe('Tests for settings component', () => {
   beforeEach(() => {
     cy.createReport();
     cy.createOtherReport();
-    cy.visit("");
-    // const desiredView = "File storage";
-    // cy.get("[data-cy-change-view-dropdown]").select(desiredView);
-    // cy.get("[data-cy-change-view-dropdown] option:selected").should(
-    //   "have.text",
-    //   desiredView
-    // );
+    cy.visit('');
   });
 
-  xit("should alter spacing when spacing setting is altered", () => {
-    cy.get("[data-cy-open-settings-modal]").as("openSettingsModal").click();
-    cy.get("[data-cy-spacing-dropdown-input]").select("1x");
-    cy.get("[data-cy-spacing-dropdown-input] option:selected").should(
-      "have.text",
-      "1x"
+  it('should alter spacing when spacing setting is altered', () => {
+    cy.get('[data-cy-open-settings-modal]').as('openSettingsModal').click();
+    cy.get('[data-cy-spacing-dropdown-input]').select('1x');
+    cy.get('[data-cy-spacing-dropdown-input] option:selected').should(
+      'have.text',
+      '1x',
     );
-    cy.get("[data-cy-settings-modal-save-changes]").as("saveButton").click();
+    cy.get('[data-cy-settings-modal-save-changes]').as('saveButton').click();
     cy.get('[data-cy-record-table-index="0"]')
-      .find("td")
+      .find('td')
       .first()
-      .as("tableCell");
-    cy.get("@tableCell").should("have.attr", "style", "padding: 0.25em 0px;");
-    cy.get("@openSettingsModal").click();
-    cy.get("[data-cy-spacing-dropdown-input]").select("0x");
-    cy.get("[data-cy-spacing-dropdown-input] option:selected").should(
-      "have.text",
-      "0x"
+      .as('tableCell');
+    cy.get('@tableCell').should('have.attr', 'style', 'padding: 0.25em 0px;');
```

Comment:
This comment didnt get added to the review somehow, but I wanted to know what the `@...` selector does as I havent seen that before